shell/platform/tizen/channels/settings_channel.h

```diff
@@ -27,11 +23,13 @@ class SettingsChannel {
   virtual ~SettingsChannel();
 
  private:
+#ifndef __X64_SHELL__
```

Comment:
Are you preparing to refactor the setting channel too? If so, it doesn't matter... but ,for testing purposes, Interface differences would be minimized as possible.

Reply:
No, I'm not. The settings channel isn't implemented by the x64 embedder, then how do you run tests for unimplemented methods using the x64 shell?

Reply:
Using ``MOCK_METHOD``?
I'm not testing the actual operation of the function, I'm just trying to test the interface.

Reply:
I think I somewhat understand what you're trying to do, but I still doubt if you really want to test these private methods. These methods are only there for the device embedder implementation (settings_channel.cc), and not used by the desktop embedder (settings_channel_stub.cc). You may later exclude some methods from #ifdef after adding some desktop embedder-side implementation of `SettingsChannel`.

Reply:
Right, I agree with what you're talking about. 
I guess further discussion is not relevant this PR. :smile: 
Let's discuss next individual unit test PR.

Reply:
Okay, let's talk about the matter later. Thanks!